networkx/algorithms/link_prediction.py

```diff
@@ -300,7 +300,7 @@ def common_neighbor_centrality(G, ebunch=None, alpha=0.8):
         If `G` is a `DiGraph`, a `Multigraph` or a `MultiDiGraph`.
 
     NetworkXAlgorithmError
-        If self loops exsists in `ebunch` or in `G` (if `ebunch` is `None`).
+        If self loops exists in `ebunch` or in `G` (if `ebunch` is `None`).
```

Comment:
```suggestion
        If self loops exist in `ebunch` or in `G` (if `ebunch` is `None`).
```
Should this be "exist" since self loops is plural?

Reply:
Thanks. Modified